custom_components/powercalc/data/tuya/CS3S/model.json

```
@@ -0,0 +1,17 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly Plus 1PM",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.6.6:docker"
+    },
+    "aliases": [
+        "LSC Smart Connect moodlight",
+        "3004154"
+    ],
+    "name": "Tuya CS3S/BK7231N",
```

Name is preferred to be the marketed name, as customer see it.

```suggestion
    "name": "LSC Smart Connect moodlight",
```

---

Changed